docs/tutorials/2.1/usingthymeleaf.md

```diff
@@ -1199,7 +1199,7 @@ Let's use this new syntax. Meet the `th:href` attribute:
 <a href="details.html" th:href="@{/order/details(orderId=${o.id})}">view</a>
 
 <!-- Will produce '/gtvg/order/3/details' (plus rewriting) -->
-<a href="details.html" th:href="@{/order/{orderId}/details(orderId=${o.id})}">view</a>
```

Comment:
This was actually correct. This kind of URL variable processing is a Spring feature, mirrored by thymeleaf here.